src/main/java/cashleh/transaction/Income.java

```diff
@@ -0,0 +1,12 @@
+package cashleh.transaction;
+
+public class Income extends Transaction {
+
+    public Income(String description, double amount) {
+        super(description, amount);
+    }
+    @Override
+    public String toString() {
+        return getDescription() + " (amount: " + getAmount() + ")";
```

Comment:
Could maybe show the amount first followed by the description and also add a "+" before the amount so that the user immediately sees that he/she is indeed shown the incomes.
e.g. "+ $2,500: Monthly Salary for October"